lib/fizzy/parser.cpp

```diff
@@ -174,17 +174,13 @@ inline parser_result<ConstantExpression> parse_constant_expression(
         break;
     }
     case Instr::f32_const:
-        // TODO: support this once floating points are implemented
         result.kind = ConstantExpression::Kind::Constant;
-        result.value.constant = 0;
-        pos = skip(4, pos, end);
```

Comment:
The `skip()` function is probably unused now.

Reply:
Removed.